snow/validators/logger.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package validators
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"go.uber.org/zap"
+)
+
+var _ NodeStakeLogger = (*logger)(nil)
+
+// NodeStakeLogger allows logging stake changes of a specified validator
+type NodeStakeLogger SetCallbackListener
+
+type logger struct {
+	subnetID ids.ID
+	nodeID   ids.NodeID
+	log      logging.Logger
+}
+
+func NewLogger(subnetID ids.ID, targetNode ids.NodeID, log logging.Logger) NodeStakeLogger {
+	return &logger{
+		subnetID: subnetID,
+		nodeID:   targetNode,
+		log:      log,
+	}
+}
+
+func (l *logger) OnValidatorAdded(valID ids.NodeID, _ *bls.PublicKey, txID ids.ID, weight uint64) {
```

Comment:
Super nit-picky, but I think we changed the naming from `valID` / `vdrID` / `validatorID` to just be `nodeID` in this package.